demos/notebooks/demo_dendritic.ipynb

```diff
@@ -164,11 +201,27 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "#%% start a cluster for parallel processing (if a cluster already exists it will be closed and a new session will be opened)\n",
-    "if 'dview' in locals():\n",
+    "print(f\"You have {psutil.cpu_count()} CPUs available in your current environment\")\n",
+    "processors_to_use = psutil.cpu_count() - 4 #  None  # Set to number less than cpu_count - 1 if you have memory problems"
```

Comment:
what happens if psutil.cpu_count() returns 1?

Reply:
@pgunn thanks for all this very helpful: That `psutil.cpu_count() - 4` was meant to be temporary while I worked on it. I will set it back to `None`.